tests/hook_main.py

```diff
@@ -2609,6 +2610,72 @@ def faq_get_list(transaction):
         db.session.commit()
 
 
+# ------------------------- FAQ Types -------------------------
+@hooks.before("FAQ Types > FAQ Type Collection > Create FAQ Type")
+def faq_type_post(transaction):
+    """
+    POST /faqs
```

Comment:
Please correct the link in the docstrings.

Reply:
Resolved